contracts/protocol/bases/TwinBase.sol

```diff
@@ -138,7 +139,7 @@ contract TwinBase is ProtocolBase, IBosonTwinEvents {
         twin.tokenType = _twin.tokenType;
 
         // Notify watchers of state change
-        emit TwinCreated(twinId, sellerId, _twin, sender);
+        emit TwinCreated(twin.id, sellerId, _twin, sender);
```

Comment:
This should stay `twinId` as it was to avoid reading from storage.

Reply:
fixed